ui/src/app/components/workflows/workflow/workflow.component.ts

```diff
@@ -123,19 +139,32 @@ export class WorkflowComponent implements OnInit, OnDestroy {
   }
 
   createWorkflow() {
-    this.confirmationDialogServiceSubscription = this.confirmationDialogService
-      .confirm(ConfirmationDialogTypes.YesOrNo, texts.CREATE_WORKFLOW_CONFIRMATION_TITLE, texts.CREATE_WORKFLOW_CONFIRMATION_CONTENT)
-      .subscribe((confirmed) => {
-        if (confirmed) this.store.dispatch(new CreateWorkflow());
-      });
+    this.showHiddenParts();
```

Comment:
Maybe you could show hidden parts only if form was not valid? If the form is valid and the user presses cancel, the appearance will have changed for no apparent reason
Of course, you may also remove the confirmation box altogether here ;)

Reply:
Fixed first part. I see you dont like confirmation box, but I will keep it there :D